packages/shared/src/components/modals/RanksModal/index.tsx

```diff
@@ -39,11 +41,31 @@ export default function RanksModal({
         <ModalCloseButton onClick={onRequestClose} />
       </ModalHeader>
       <IntroSection onShowAccount={onShowAccount} user={user} />
-      <RanksBadges rank={currentRank} progress={progress} />
-      <RanksTags tags={tags} />
-      {!hideProgress && reads > devCardLimit && (
-        <DevCardFooter rank={rank} user={user} />
+      <RanksBadges rank={rank} progress={progress} />
+      {!user && (
+        <div className="flex flex-col items-center mt-2">
+          <span className="typo-footnote">
+            Sign up to add your weekly achievements to your profile.
```

Comment:
I think this should also be based on the rank color right?

Reply:
I think this should be good, in the design, it was a gold rank but had the text in primary.

Reply:
You're right!
My mind played tricks on me

Reply:
Lol, many things having their own variations for color 😆